README.md

```diff
@@ -109,13 +109,37 @@ GOOGLE_TASK_SHEET_ID=AIzaSyAChRujfXXXXXXXXMP5eouRRQ6bxV-1u_o
 ```
 
 
+## Important Changes and Dictionary Sheets
+
+In the current workflow, 2 Google Docs spreadsheets are used to keep track of 'important changes' and 'dictionary' records. The UI at this time posts annotations to DB with these attributes and also appends records to those spreadsheets. To get this working:
+1) Make copies of these spreadsheets:
+* [Important Changes](https://docs.google.com/spreadsheets/d/1S2mZKuV2v7-uec2eGA0zp3X1v1IAnoCHkLXmL-ChqnM/edit#gid=1804226491)
+
+* [Dictionary](https://docs.google.com/spreadsheets/d/1YRo1uNRRX92eSo2JiGEu50TpPVXYaLchrUVOZ3UW0Bs/edit#gid=554811086)
```

Comment:
We need to either make publicly viewable versions of these sheets for people to copy or detail their structure as with the tasking sheet.